lib/messaging/messagehandler.rb

```diff
@@ -144,13 +157,13 @@ def send_message_obj_to_individual_handlers(message_obj)
       if message_obj.handled == false
         handler.handle_message( message_obj )
       end
-    end     
+    end
   end
 
   def check_if_message_handled(message_obj)
     if message_obj.handled == false
       @dbaccess.write_to_log(2,'message could not be handled')
-      send_error(sender, '', 'message could not be handled')
+      send_error(message_obj.sender, '', 'message could not be handled')
```

Comment:
This may have been a typo during the refactor, as the variable `sender` was `nil`.